scripts/getDivviIntegrators.ts

```diff
@@ -0,0 +1,168 @@
+import { writeFileSync } from 'fs'
+import yargs from 'yargs'
+import { Address, encodeEventTopics } from 'viem'
+import { LogField } from '@envio-dev/hypersync-client'
+import { paginateQuery } from './utils/hypersyncPagination'
+import { getHyperSyncClient } from './utils'
+import { NetworkId } from './types'
+import { divviRegistryAbi } from '../abis/DivviRegistry'
+import { rewardPoolAbi } from '../abis/RewardPool'
+
+const DIVVI_REGISTRY_CONTRACT_ADDRESS =
+  '0xEdb51A8C390fC84B1c2a40e0AE9C9882Fa7b7277'
+const DIVVI_INTEGRATION_REWARDS_ENTITY =
+  '0x6226ddE08402642964f9A6de844ea3116F0dFc7e'
+
+async function getArgs() {
+  const argv = await yargs
+    .env('')
+    .option('output-file', {
+      alias: 'o',
+      description: 'output file',
+      type: 'string',
+    })
+    .option('reward-pool', {
+      description: 'RewardPool contract address',
+      type: 'string',
+      demandOption: true,
+    })
+    .option('reward-amount', {
+      description: 'reward amount for integration',
+      type: 'string',
+    }).argv
```

Comment:
should we allow passing in the start and end blocks, to avoid doing extra work between script executions?